ipaserver/install/ipa_restore.py

```diff
@@ -687,8 +688,9 @@ def bak2db(self, instance, backend, online=True):
             logger.info("Waiting for restore to finish")
             wait_for_task(conn, dn)
         else:
-            args = [paths.BAK2DB,
-                    '-Z', instance,
+            args = [paths.DSCTL,
```

Comment:
``dsctl`` is not compatible with ``if backend is not None: args.append('-n')``

Reply:
You're right. Not caught because bak2db isn't used anywhere. If I remember correctly I added it as a parallel to the db2bak call (belt and suspenders with backup). I'll fix it up.